libsolidity/interface/ABI.cpp

```diff
@@ -26,12 +26,31 @@ using namespace std;
 using namespace dev;
 using namespace dev::solidity;
 
+namespace
+{
+bool containsStoragePointer ( TypePointers const& _pointers )
```

Comment:
I think the defining feature of this variable is not that it is a list of pointers but what the pointers point to.

Reply:
Also the function should probably not use the term `Pointer` because that might lead to the assumption that storage references would not trigger the check.